Frameworks/Social.Xaml/ShareSheet.xaml.cpp

```diff
@@ -0,0 +1,111 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "pch.h"
+#include <wrl/client.h>
+#include <wrl/implements.h>
+#include <ppltasks.h>
+#include "ShareSheet.xaml.h"
+#include "ShareSheetControl.h"
+#include "ErrorHandling.h"
+
+using namespace Social::Xaml;
+using namespace Microsoft::WRL;
+using namespace Windows::Foundation;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Media;
+using namespace Windows::UI::Xaml::Media::Imaging;
+using namespace Windows::Web::Http;
+using namespace concurrency;
+
+ShareSheet::ShareSheet(Platform::String^ token) {
+    InitializeComponent();
+    _token = token;
+}
+
+void ShareSheet::ShareSheet_Post(ContentDialog^ sender, ContentDialogButtonClickEventArgs^ args) {
+    // Rudimentary implementation of posting logic
```

Comment:
nit: maybe more placeholder-ish comments then?

---
In reply to: [107996694](https://github.com/Microsoft/WinObjC/pull/2314#discussion_r107996694) [](ancestors = 107996694,107996496)